packages/envied_generator/lib/src/generator.dart

```diff
@@ -31,6 +31,23 @@ final class EnviedGenerator extends GeneratorForAnnotation<Envied> {
     ConstantReader annotation,
     BuildStep buildStep,
   ) async {
+    final enviedAnnotations = element.metadata
+        .where((a) => a.element?.displayName == 'Envied')
+        .map((e) => ConstantReader(e.computeConstantValue()));
+    var generatedClassesAltogether = StringBuffer();
```

Comment:
Please make this `final` instead of using a `var` and declare it with an explicit type.